README.md

```diff
@@ -280,7 +280,7 @@ Compose.prototype.map = function(f) {
 #### `traverse` method
 
 ```hs
-traverse :: Apply f, Traversable t => t a ~> ((a -> f b), (c -> f c)) -> f (t b)
+traverse :: Apply f, Traversable t => t a ~> (a -> f b, c -> f c) -> f (t b)
```

Comment:
This is unambiguous since the only meaning of `,` in the grammar is "argument separator".